core/src/main/resources/hudson/model/Job/configure.jelly

```diff
@@ -30,6 +30,15 @@ THE SOFTWARE.
 
   <l:layout norefresh="true" type="one-column" permission="${it.EXTENDED_READ}" title="${%Config(it.displayName)}">
 
+    <j:choose>
+      <j:when test="${app.hasPermission(it.CONFIGURE)}">
```

Comment:
https://issues.jenkins-ci.org/browse/JENKINS-61321 / https://github.com/jenkinsci/jenkins/pull/4541

Reply:
🤕, tested it with an admin user and read only, but not a lower privileged but still configure -.-